inc/theme-options.php

```diff
@@ -539,6 +558,17 @@ function odm_get_legal_disclaimer()
     }
 }
 
+function odm_get_post_types_for_category_page()
+{
+    $options = get_option('odm_options');
+    if (isset($options['category_page'])) {
+        $post_type = array_map('trim', explode(',', $options['category_page']));
+        return $post_type;
+    } else {
+        return false;
```

Comment:
@Huyeng I would return a default value in case this options has not being set. For that, I would change this line to ```return array('news-article', 'topic', 'profile', 'announcement', 'site-update', 'map-layer');```

Do you agree?